packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20231123</version>
+    <version>0.0.0.20241220</version>
```

Comment:
I think you have used an old base for this commit, as the old version is quite old. Please rebase so that GH allows us to merge this PR. 😉